### PR TITLE
feat: add default logo links for Ansys and PyAnsys logos

### LINKED ISSUE #467: PyAnsys and Ansys Logo Redirection
**Current Behavior**: Clicking on the logo does not redirect users to the home page.

**Desired Change**: The logo should redirect users to the home page when clicked.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
fix #467

### CODE REVIEW

File: doc/source/user-guide/configuration.rst [hunk 1]
```diff
@@ -54,6 +54,20 @@ To use the logo in both dark and light modes, add the following code to the``htm
              "logo": "no_logo",
          }
 
+.. note::
+
+    By default, if ``ansys`` logo is displayed, the logo links to the Ansys website. If the PyAnsys logo is displayed, the logo links to the PyAnsys website.
+    if you want to change the link, you can set the ``logo_link`` option in the ``conf.py`` file.
```
Comment (SMoraisAnsys) [inline doc/source/user-guide/configuration.rst:60]:
```suggestion
    If you want to change the link, you can set the ``logo_link`` option in the ``conf.py`` file.
```

File: src/ansys_sphinx_theme/__init__.py [hunk 2]
```diff
@@ -377,8 +380,18 @@ def configure_theme_logo(app: Sphinx):
 
     if logo_option == "ansys":
         theme_options["logo"] = ansys_logo
+        theme_options["logo_link"] = (
+            ANSYS_LOGO_LINK
+            if not theme_options.get("logo_link")
+            else theme_options.get("logo_link")
+        )
```
Comment (SMoraisAnsys) [inline src/ansys_sphinx_theme/__init__.py:387]:
```suggestion
        theme_options["logo_link"] = theme_options.get("logo_link", ANSYS_LOGO_LINK)
```

File: src/ansys_sphinx_theme/__init__.py [hunk 3]
```diff
@@ -377,8 +380,18 @@ def configure_theme_logo(app: Sphinx):
...
     elif logo_option == "pyansys":
         theme_options["logo"] = pyansys_logo
+        theme_options["logo_link"] = (
+            PYANSYS_LOGO_LINK
+            if not theme_options.get("logo_link")
+            else theme_options.get("logo_link")
+        )
```
Comment (SMoraisAnsys) [inline src/ansys_sphinx_theme/__init__.py:394]:
```suggestion
        theme_options["logo_link"] = theme_options.get("logo_link", PYANSYS_LOGO_LINK)
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left minor comments

File: src/ansys_sphinx_theme/__init__.py (hunk 2 above)
Comment (SMoraisAnsys) [reply src/ansys_sphinx_theme/__init__.py:387]: After some though, did you do the current implementation to handle the case where `theme_options["logo_link"]=""` ?

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**